### ISSUE TITLE
Implementation details about batch size and learning rate

### ISSUE THREAD

**Issue by asheroin** [BadourAlBahar/pose-with-style#1]
I have noticed that for the first training phase, batch size was set to 1 and thus the traning time would explose to very huge. Is it because some experiment had been conducted and found that a big (or, just 8) batch size would lead to a vary bad result? It would be very pleasure if you could share the conclution about this.

By the way, I found that the learning rate for D and G was scaling by some strange number, especially the 16/17 for D. Since it is an infinite decimal, I just confused about such settings. I mean, for code simplicity, people would tend to chose some number like 0.5, 0.8, or 0.2, etc. Are there some interesting conclutions about such wired number?

**Comment by BadourAlBahar** [comment]
At the time of running the first phase of training, I only had access to 8 NVIDIA 2080 Ti GPUs and I could only fit a batch size of 1 per GPU. I am not sure how the results would differ with different training settings.
For the learning rates, we followed [Rosinality: StyleGAN 2 in PyTorch](https://github.com/rosinality/stylegan2-pytorch).
